Main.java

```diff
@@ -9,7 +10,25 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // create a Scanner for user input
+    Scanner input = new Scanner (System.in);
+     //declare a variable to store the first numbers
+     System.out.println("Please enter an integer:");
+     // prompt the user for the first number 
+     int A = input.nextInt();
+     //declare a variable to store second number
+     //prompt the user for the second number
+     
+     System.out.println ("Please enter another integer:");
+     int B = input.nextInt();
```

Comment:
Remember, variables should always be named using camelCase and describe what they are storing or being used for. A and B would be poor choices for variables here and should start with a lowercase letter.